static/app/views/releases/detail/index.tsx

```diff
@@ -249,54 +251,18 @@ class ReleasesDetail extends DeprecatedAsyncView<Props, State> {
 // ========================================================================
 // RELEASE DETAIL CONTAINER
 // ========================================================================
-
 type ReleasesDetailContainerProps = Omit<Props, 'releaseMeta'>;
-type ReleasesDetailContainerState = {
-  releaseMeta: ReleaseMeta | null;
-} & DeprecatedAsyncComponent['state'];
-class ReleasesDetailContainer extends DeprecatedAsyncComponent<
-  ReleasesDetailContainerProps,
-  ReleasesDetailContainerState
-> {
-  shouldReload = true;
+function ReleasesDetailContainer(props: ReleasesDetailContainerProps) {
+  const params = useParams<{release: string}>();
```

Comment:
```suggestion
  const {release} = useParams<{release: string}>();
```
super nit, to avoid doing `params.release` everywhere else